aws/chamber/main.tf

```diff
@@ -28,6 +28,12 @@ variable "account_id" {
   description = "AWS account ID"
 }
 
+variable "parameter_groups" {
+  type        = "string"
```

Comment:
This should use the native type of `list`

Reply:
You can use JSON notation in envs. e.g. `["one", "two"]`

Reply:
then it would be hard to manage with ENV vars